src/main/java/ch/epfl/biop/wrappers/cellpose/Cellpose.java

```diff
@@ -79,19 +80,39 @@ static void execute(List<String> options, Consumer<InputStream> outputHandler)
 
         // input options
         cmd.addAll(options);
-        //
-        System.out.println( cmd );
```

Comment:
Why not IJ.log ? Did you test it on Fiji ? Where is this written ?

Reply:
it's printed in the console.
I "prefer" to print in the console (to print in the log) because now I enforce --verbose and someone might want to use the log for something else in a macro/script...

Reply:
Ok, then all good, you can merge it